.eslintrc.json

```diff
@@ -15,6 +15,8 @@
   ],
   "globals": { "BigInt": true, "console": true, "WebAssembly": true },
   "rules": {
+    "@typescript-eslint/no-useless-catch": "off",
+    "@typescript-eslint/no-empty-function": "off",
```

Comment:
having a really hard time with eslint-disable comments for some reason. Disabling the rule globally doesn't seem to help either